### PR TITLE
Ft initialize project

### PR DESCRIPTION
**Pull request by freddykaberuka**
## Completed Task

- Add Book component
- Add Category component
- Add Navigation Bar
- add license
- update readme

##ScreenShoot

Screen Shot 2023-02-25 at 14 47 40

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Deepakdanger) [inline src/App.js:2]: - [ ] Kindly remove this & solve the linter manually

File: src/components/Book.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Deepakdanger) [inline src/components/Book.js:2]: - [ ] Kindly remove this & solve the linter manually

File: src/components/nav.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Deepakdanger) [inline src/components/nav.js:2]: - [ ] Kindly remove this & solve the linter manually

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @freddykaberuka , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand:  @freddykaberuka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :green_circle: 

- PR is professional :+1: 
- Linter checks passed :heavy_check_mark: 

## Optional suggestions
**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**